client/client.go

```diff
@@ -162,29 +165,45 @@ func (c *Client) Authenticate() error {
 }
 
 func (c *Client) Run() {
+	inventoryTicker := time.NewTicker(c.Config.InventoryInterval)
+	updateTicker := time.NewTicker(c.Config.UpdateInterval)
+
+auth:
```

Comment:
This is the very first time I se `goto` in golang :exploding_head:

Reply:
it was the cleanest way to "restart" the client without making the code more complicated